web/js/components/animation-widget/play-queue.js

```diff
@@ -392,9 +394,12 @@ class PlayAnimation extends React.Component {
    * @param index {string} Date string
    */
   animate(index) {
-    const { selectDate, endDate, speed, isPlaying } = this.props;
+    const {
+      selectDate, endDate, speed, isPlaying,
+    } = this.props;
     let currentDateStr = index;
-    let nextDateStr, nextDateParsed;
+    let nextDateStr; let
```

Comment:
`let nextDateStr, nextDateParsed;` seems better than `let nextDateStr; let nextDateParsed;`
I'd prefer either prevent same line declarations out right or allow  `let nextDateStr, nextDateParsed;`

Reply:
Yeah in this particular instance I think the autofix wasn't able to handle the formatting properly.  It should be one var per line. This wasn't intentional.